src/server.cpp

```diff
@@ -552,12 +568,12 @@ inline void CServer::connectChannelSignalsToServerSlots()
 template<>
 inline void CServer::connectChannelSignalsToServerSlots<0>() {}
 
-void CServer::CreateAndSendJitBufMessage ( const int iCurChanID,
+void CServer::CreateAndSendJitBufMessage ( const int iCurChanID, 
```

Comment:
More trailing whitespace.  Could you review all the changes before committing to avoid this, please.